src/olympia/addons/tests/test_models.py

```diff
@@ -1820,7 +1820,7 @@ def test_current_version_is_listed_only(self):
 
     def test_firefox_versions(self):
         self.setup_new_version()
-        assert self.addon.update_version()
+        self.addon.update_version()
```

Comment:
With `latest_version` no longer set by `update_version()`, that assertion (which wasn't very useful in that test) will fail.